pinc/graph_data.inc

```diff
@@ -857,3 +838,16 @@ function query_graph_cache($function, $args = [], $expiration = 3600)
 {
     return memoize_function($function, $args, $expiration, get_desired_language());
 }
+
+/**
+ * Given an array of numbers and a window, calculate the moving average over
+ * the window.
+ */
+function calculate_sma(array $array, int $period): array
+{
+    $results = [];
+    for ($i = 0; $i < (count($array) - $period); $i++) {
```

Comment:
I think that `<` should be `<=`, otherwise you're skipping the window that ends at the end of the array.

(This might be why the example graphs show no SMA for the last day. Mind you, they also show no Goal for the last day, which presumably can't be pinned on this bug.)